db/migrate/20220810093522_add_user_ref_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
```

Comment:
- [ ] Here, you need to use the alias `author` instead of user as specified in the ERD diagram. Kindly do something like the following to implement this.
```ruby
 add_reference :posts, :author, null: false, foreign_key: { to_table: 'users' }
```

Reply:
Thanks @zhadier I changed it